test/quantization/core/test_workflow_ops.py

```diff
@@ -1064,10 +1072,13 @@ def test_fused_obs_fake_quant_moving_avg(self, device, symmetric_quant, use_bool
         Tests the case where we call the fused_obs_fake_quant op multiple times
         and update the running_min and max of the activation tensors.
         """
-        in_running_min_ref = out_running_min_ref = float("inf")
-        in_running_min_op = torch.tensor(float("inf"), device=device)
-        in_running_max_ref = out_running_max_ref = float("-inf")
-        in_running_max_op = torch.tensor(float("-inf"), device=device)
+        sampled_dtype = st.sampled_from(["bf16", "fp32"]) if device == "cuda" else "fp32"
```

Comment:
I think we should avoid using `sampled_from` where possible to reduce potential non-determinism. Can we parametrize on `[bf16, fp32]` instead?